pkg/asset/agent/image/agentimage.go

```diff
@@ -87,6 +89,21 @@ func (a *AgentImage) fetchAgentTuiFiles(releaseImage string, pullSecret string,
 		if err != nil {
 			return nil, err
 		}
+
+		// Rename libnmstate shared lib
+		matched, err := filepath.Match(filepath.Base(libnmstateFull), filepath.Base(f))
+		if err != nil {
+			return nil, err
```

Comment:
In general we should consider [wrapping these errors](https://pkg.go.dev/github.com/pkg/errors#Wrapf) to make it easier to discern failure points within the same function